tinkerbell/stack/values.yaml

```diff
@@ -6,24 +6,25 @@ stack:
     type: LoadBalancer
   selector:
     app: tink-stack
-  loadBalancerIP: 192.168.2.112
+  loadBalancerIP: &publicIP 192.168.2.112
```

Comment:
Is this value correct? What does it do? Fall back to .112 when publicIP is not given?

I see that the .112 was replaced with *publicIP below (https://github.com/tinkerbell/charts/pull/58/files#diff-cb32831c03997bafdbf931acb3be6ed9a69aee4459081005c8a2cd2df12ba243R60-R65).

Reply:
I'm curious why we're using `192.168.2` as well? Wouldn't a more typical network like `192.168.0` be appropriate?

Reply:
This will almost certainly need to be changed by users regardless. If you want i will change it to `192.168.0.112`, let me know.

Another idea could be to set this value to an incorrect/bad value and add a check in the helm chart some how to make sure folks set it appropriate to their environments.

Reply:
That's a good idea so people don't get accidental errors.